File: Program5.java
import java.lang.*;
import java.util.*;


class Arithmatic
{
    private int iNo1;
    private int iNo2;

    public Arithmatic(int iValue1, int iValue2)
    {
        iNo1 = iValue1;
        iNo2 = iValue2;
    }

    public int Addition()
    {
        int iAns = 0;
        iAns = iNo1 + iNo2;
        return iAns;
    }

}

class Program5
{
    public static void main(String arg[])
    {
       int iValue3 = 0;

       Scanner sobj = new Scanner(System.in);

       System.out.println("Enter first number");
       int iValue1 = sobj.nextInt();

       System.out.println("Enter second number");
       int iValue2 = sobj.nextInt();

       Arithmatic aobj = new Arithmatic(iValue1, iValue2);

       iValue3 = aobj.Addition();

       System.out.println("Addition is: "+iValue3);

    }
}

File: Program5.cpp
#include<iostream>
using namespace std;

class Arithmatic
{
    private:
       int iValue1;
       int iValue2;
    
    public:
       Arithmatic(int iNum1, int iNum2)
       {
           iValue1 = iNum1;
           iValue2 = iNum2;
       }
       
       int Addition()
       {
           int iAns = 0;
           iAns = iValue1 + iValue2;
           return iAns;
       }
};

int main()
{
    int iNo1 = 0;
    int iNo2 = 0;
    int iNo3 = 0;

    cout<<"Enter first number"<<endl;
    cin>>iNo1;

    cout<<"Enter second number"<<endl;
    cin>>iNo2;
    
    Arithmatic aobj(iNo1, iNo2);

    iNo3 = aobj.Addition();

    cout<<"Addition is:"<<iNo3<<endl;
     
    return 0;
}